src/chapters/strings/encoding-characters.rst

```diff
@@ -0,0 +1,119 @@
+Encoding Characters
+===================
+
```

Comment:
OK, wow.  There is a lot to unpack here.

First a disclaimer - I find the whole why-use-binary-binary-vs.-decimal-convert-between, etc. stuff really cool.  "How does the computer store 'LaunchCode'?"  Neat stuff, but kinda dry.

As I channeled my inner novice coder, my eyes glazed over by the second paragraph.

I can see skipping right to ASCII encoding without a significant loss of clarity.  Do we really need to use binary to preface character encoding?  Is this an example of a "nice to know" topic or a "need to know"?

We may want to poll the ed team for other opinions.

Reply:
I have mixed feelings on it as well, but at the end of the day it seems like even new developers should know that all data is stored in binary format. What about moving some of the content into an appendix?

Reply:
I like the idea of shifting the binary discussion to the appendix.  We can precede it with a BRIEF, engaging intro and then move into the more relevant ASCII encoding.

"Computers do not store data (like the character 'A') in a form that is easily readable by humans.  Instead, the data is changed into **binary** format.  Click *here* for a detailed romp through this cool but kinda dry topic."

(Feel free to edit the above as you see 01100110 01101001 01110100 ).